doc/API/Switching.md

```diff
@@ -69,3 +69,81 @@ Output:
   ]
 }
 ```
+
+### `list_links`
+
+Get a list of all or per hostname discovered Links.
+
+Route: `/api/v0/resources/links`
+
+Input:
+
+  - hostname = Either the devices hostname or id
+
+Example:
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/resources/links
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/resources/links?hostname=host.example.com
```

Comment:
I wonder if this route should be `/api/v0/devices/host.example.com/links`?

Reply:
Yes resonable, i'll prepare it in follow-up commit.

Reply:
Can you do it in this one, no point in releasing something that will change, especially if people start using it.

Reply:
Yes sure in this PR. Just added as separate call.